WordPress/Classes/ViewRelated/Site Creation/Shared/ErrorStates/InlineErrorRetryTableViewCell.swift

```diff
@@ -0,0 +1,142 @@
+
+import UIKit
+
+import Gridicons
+import WordPressShared
+
+// MARK: - InlineErrorRetryTableViewCellAccessoryView
+
+/// The accessory view comprises a retry Gridicon & advisory text
+///
+private class InlineErrorRetryTableViewCellAccessoryView: UIStackView {
+
+    // MARK: Properties
+
+    /// A collection of parameters uses for view layout
+    private struct Parameters {
```

Comment:
Nitpick: Same comment as on the verticals PR about perhaps renaming this – `Metrics` or similar?

Reply:
I have pushed a commit that addresses this.